pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -130,17 +130,17 @@ type ServiceClass struct {
 	ExternalID string `json:"externalID"`
 
 	// OSB-specific
-	OSBTags                    []string `json:"osbTags,omitempty"`
-	OSBRequires                []string `json:"osbRequires,omitempty"`
-	OSBMaxDBPerNode            *string  `json:"osbMaxDBPerNode,omitempty"`
-	OSBDashboardOAuth2ClientID *string  `json:"osbDashboardOAuth2ClientID,omitempty"`
-	OSBDashboardSecret         *string  `json:"osbDashboardSecret,omitempty"`
-	OSBDashboardRedirectURI    *string  `json:"osbDashboardRedirectURI,omitempty"`
+	Tags                    []string `json:"tags,omitempty"`
+	Requires                []string `json:"requires,omitempty"`
+	MaxDBPerNode            *string  `json:"maxdbpernode,omitempty"`
+	DashboardOAuth2ClientID *string  `json:"dashboardOAuth2ClientID,omitempty"`
+	DashboardSecret         *string  `json:"dashboardSecret,omitempty"`
+	DashboardRedirectURI    *string  `json:"dashboardRedirectURI,omitempty"`
 	// Description is a short description of the service.
 	Description string `json:"description"`
 
 	// Metadata fields
-	OSBMetadata *runtime.RawExtension `json:"osbMetadata, omitempty"`
+	Metadata *runtime.RawExtension `json:"externalMetadata, omitempty"`
```

Comment:
`ServiceClass` already has a `metadata` field from the kubernetes `ObjectMeta`. I pseudocopied the `ExternalID` idea and added an `External` prefix. This was after I had renamed the field itself to `Metadata`.